src/providers/signature.ts

```diff
@@ -86,7 +93,7 @@ function getCurrentArgsRanges(document: TextDocument, idx: number): Range[] {
         (previousRangeIndex > 1 &&
             ['->', 'some->'].includes(previousFunction)) ||
         (previousRangeIndex > 1 &&
-            previousRangeIndex % 2 &&
+            previousRangeIndex % 2 !== 0 &&
```

Comment:
Was this check for the modulus result just missing before?

Reply:
yep! it worked before because 0 is false-y. however, we're assigning it to a boolean variable and if it was 0 then `isInThreadFirst`'s value would be 0 instead of false.